x/examples/website-wrapper-app/wrapper_app_project/template/ios/App/App/AppDelegate.swift

```diff
@@ -40,16 +33,52 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
 
-    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+    func application(
+        _ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]
+    ) -> Bool {
         // Called when the app was launched with a url. Feel free to add additional processing here,
         // but if you want the App API to support tracking app url opens, make sure to keep this call
         return ApplicationDelegateProxy.shared.application(app, open: url, options: options)
     }
 
-    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
+    func application(
+        _ application: UIApplication, continue userActivity: NSUserActivity,
+        restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void
+    ) -> Bool {
         // Called when the app was launched with an activity, including Universal Links.
         // Feel free to add additional processing here, but if you want the App API to support
         // tracking app url opens, make sure to keep this call
-        return ApplicationDelegateProxy.shared.application(application, continue: userActivity, restorationHandler: restorationHandler)
+        return ApplicationDelegateProxy.shared.application(
+            application, continue: userActivity, restorationHandler: restorationHandler)
+    }
+
+    @discardableResult
+    private func tryProxy() -> Bool {
+        var dialerError: NSError?
+        if let dialer = MobileproxyNewSmartStreamDialer(
+            MobileproxyNewListFromLines(Config.domainList),
+            Config.smartDialer,
+            MobileproxyNewStderrLogWriter(),
+            &dialerError
+        ) {
+            if let _ = dialerError {
+                return false
+            }
+
+            var proxyError: NSError?
+            self.proxy = MobileproxyRunProxy(
+                "127.0.0.1:0",
+                dialer,
+                &proxyError
+            )
+            
+            Config.proxyPort = String(self.proxy?.port() ?? 0)
+
+            if let _ = proxyError {
```

Comment:
Same as `dialerError`?

```suggestion
            if proxyError {
```